Lib/test/string_tests.py

```diff
@@ -217,6 +217,25 @@ def test_find(self):
                 if loc != -1:
                     self.assertEqual(i[loc:loc+len(j)], j)
 
+        # test tuple arguments
+        self.checkequal(2, '__aa__bb__', 'find', ('aa', 'bb'))
+        self.checkequal(2, '__aa__bb__', 'find', ('bb', 'aa'))
+        self.checkequal(-1, '__aa__bb__', 'find', ('cc', 'dd'))
+        self.checkequal(-1, '__aa__bb__', 'find', ())
+        self.checkequal(6, '__aa__bb__', 'find', ('aa', 'bb'), 3)
+        self.checkequal(-1, '__aa__bb__', 'find', ('aa', 'cc'), 3)
+        self.checkequal(2, '__aa__bb__', 'find', ('aa', 'bb'), 0, 10)
+        self.checkequal(-1, '__aa__bb__', 'find', ('aa', 'bb'), 0, 3)
+        self.checkequal(2, '__aa__bb__', 'find', ('aa', 'bb'), 0, 4)
+        s = '_' * 9998 + 'aaaa' + '_' * 9998
+        self.checkequal(9998, s, 'find', ('aaaa', 'bb'))
+        self.checkequal(2, 'foobar', 'find', ('ob', 'oba'))
+        self.checkequal(1, 'foobar', 'find', ('ob', 'oob'))
+        self.checkequal(0, '', 'find', ('',))
+        self.checkequal(2, '__abcd__', 'find', ('cd', 'ab'))
+        self.checkequal(2, '__abc__', 'find', ('bc', 'ab'))
+        self.checkequal(1, 'a' + 'b' * 10000, 'find', ('b' * 10000,))
+
```

Comment:
```suggestion
self.checkequal(1, 'ab' + 'c' * 100000, 'find', ('c' * 100000, 'b' + 'c' * 100000))
```

Reply:
Ah, too late. Yeah indentation must be included.